python/tests/rtl/test_requests_transport.py

```diff
@@ -153,7 +162,38 @@ def test_request_error(settings):
     """
     session = cast(requests.Session, Session(None, True))
     test = requests_transport.RequestsTransport(settings, session)
-    resp = test.request(transport.HttpMethod.GET, "/some/path")
+    resp = test.request(transport.HttpMethod.GET, "https://looker.sdk")
     assert isinstance(resp, transport.Response)
     assert resp.value == b"(54, 'Connection reset by peer')"
     assert resp.ok is False
+
+
+def test_retrieves_fully_qualified_url(rtp):
+    resp = rtp.request(
+        transport.HttpMethod.GET, "https://github.com/looker-open-source/sdk-codegen"
+    )
+    assert isinstance(resp, transport.Response)
+    assert resp.ok
+    if resp.ok:
+        content = str(resp.value)
+        assert "One SDK to rule them all, and in the codegen bind them" in content
+
+
+def test_relative_path_without_authenticator_is_just_base(rtp):
+    actual = rtp.make_path("/login")
+    assert actual == "https://looker.sdk/login"
+
+
+def test_relative_path_with_authenticator_is_api_path(rtp):
+    actual = rtp.make_path("/login", mock_auth)
+    assert actual == "https://looker.sdk/api/3.1/login"
+
+
+def test_full_path_without_auth_is_just_full_path(rtp):
+    actual = rtp.make_path("https://looker.sdk/login")
+    assert actual == "https://looker.sdk/login"
+
+
+def test_full_path_with_auth_is_just_full_path(rtp):
+    actual = rtp.make_path("https://looker.sdk/login", mock_auth)
+    assert actual == "https://looker.sdk/login"
```

Comment:
I think this is testing too deeply into the internals of the implementation (especially since I think `make_path()` should go away). I think the `test_retrieves_fully_qualified_url` is sufficient.